Doc/reference/datamodel.rst

```diff
@@ -2590,6 +2591,19 @@ through the object's keys; for sequences, it should iterate through the values.
    the container.
 
 
+.. method:: object.keys(self)
+
+   This method can be used by Python in arbitrary circumstances to treat an
+   arbitrary instance as a mapping or not.  The method should return an object
+   that can iterate over all the keys of the mapping.
+
+   .. impl-detail::
+
+      In CPython, the method is used by the ``**expression`` syntax in
```

Comment:
any identifier works certainly, but other places in the file it says `**keywords`, `**kwds` and `**kwargs`. For those used to python, something like `**kwargs` is even easier to read since it fits a very well known pattern, maybe?

Reply:
`**kwargs` can mean two things, either something you write in a function _signature_, also known as a variadic keyword parameter or a keyword catch-all, or you can write it in a function _call_. In the former case it takes an identifier and you can onely have one, and in the latter it takes any expression and you can have several of them in the same call.
I think it was written as `**expression` in the Calls section of the doc (which I link to), so I think this formulation is less ambiguous.

Reply:
From the datamo page, since you talked about other examples :
- The first occurrence is `**keywords` which in contexts designates a function signature, specifically not the unary `**` operator I'm talking about in this PR
- The second is `__init_subclass__` taking a `**kwargs` in its signature so same issue, even though it calls the super with the visually identical `**kwargs` after
- Then there is `__prepare__` and the `metaclass` call, which... are a mix of the two, I guess.